open_ce/build_feedstock.py

```diff
@@ -104,6 +104,9 @@ def build_feedstock_from_command(command, # pylint: disable=too-many-arguments,
     '''
     utils.check_if_conda_build_exists()
 
+    if 'cuda' in command.build_type:
```

Comment:
Adding this check here would make it call for every build command when we are building any open-ce environment. I think this call should be moved to `build_feedstock` entry point function and then added into build_tree's constructor too so that it can be called when `build env ` command is triggered.

Reply:
@npanpaliya I moved this check into the `build_feedstock` entry point function as suggested, just before it calls `build_feedstock_from_command`. In reviewing the constructor in `build_tree.py` I am trying to follow the thinking of adding the same check in there, and where it would be appropriate to do that. (I think I only partially understand this so I will review it some more.)